.circleci/config.yml

```diff
@@ -222,7 +222,7 @@ jobs:
       - checkout
       - run:
           name: SPM Build
-          command: swift build
+          command: swift build -c release
```

Comment:
Should we just have both? A `spm-build-debug` and a `spm-build-release` jobs. But well, if this is just building the SDK, as you said, that might be covered there so I'm ok keeping only the release version, but maybe renaming the job is still worth it.

Reply:
Yeah I'll rename it.
Like I said in the description I think we cover debug builds a lot in every other job.

Reply:
Done.